operate/schema/src/main/java/io/camunda/operate/schema/IndexSchemaValidator.java

```diff
@@ -31,7 +31,7 @@ public class IndexSchemaValidator {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(IndexSchemaValidator.class);
 
-  private static final Pattern VERSION_PATTERN = Pattern.compile(".*-(\\d+\\.\\d+\\.\\d+.*)_.*");
```

Comment:
:x: This may still falsely match some (admittedly weird) rollover patterns such as `yyyy.MM.dd_HH`, right?
What if instead of using one regex on the entire index name, we try to strip the prefix and index name first, knowing that the very next thing has to be the version? That would allow for a much more precise pattern I think.

Reply:
I don't think that it will fail, as it takes [1st group found](https://github.com/camunda/camunda/blob/2d9addcc020337a4cf824ce148c98ad922ee790d/operate/schema/src/main/java/io/camunda/operate/schema/IndexSchemaValidator.java#L82). But let me try to improve it the way you propose.